README.md

```diff
@@ -23,350 +23,46 @@
 <img src="assets/github-hr.png" alt="---" />
 <br/>
 
-**k6** is a modern load testing tool, building on [our](https://k6.io/about) years of experience in the load and performance testing industry. It provides a clean, approachable scripting API, [local](https://k6.io/docs/getting-started/running-k6) and [cloud execution](https://k6.io/docs/cloud), and flexible configuration.
+**k6** is a modern load-testing tool, building on [our years of experience](https://k6.io/about) in the performance and testing industries.
+k6 is built to be simple, powerful, and extensible. Every commit to this repo aims to help create an application with **high performance and excellent developer experience**.
+Some key features include:
 
-This is how load testing should look in the 21st century.
+- **Configurable load generation.** Even lower-end machines can simulate lots of traffic.
+- **Tests as code.** Reuse scripts, modularize logic, version control, and integrate tests with your CI.
+- **A full-featured API.** The scripting API is packed with features that help you simulate real application traffic.
+- **An embedded JavaScript engine.** The performance of Go, the scripting simplicity of vanilla ECMAscript (without baggage from the wider JS ecosystem).
```

Comment:
Two points here:
- "vanilla ECMAScript" doesn't really say much. What exactly would "non-vanilla" ES be?
- I think "without baggage from the wider JS ecosystem" is an unnecessary swipe.

So we might want to rephrase this as:

> The performance of Go, with the familiarity of JavaScript.

Reply:
Yeah you're totally right. I was under the influence of a discussion from last week :-).

Reply:
The drawback of what I just said is that ECMAScript is an absolutely hideous word, hard on both eye and ear.

Reply:
I don't think we need to mention ES, unless we're discussing the language specification in the context of the features we support. Since this is a high-level overview, I think we can stick to just JavaScript. There's nothing "vanilla" about ES; or with k6/goja's implementation of it. Repeating JS doesn't bother me.

But maybe someone else can chime in.